docs/azure/sdk/azure-sdk-configure-proxy.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Configure a proxy server when using the Azure SDK for .NET 
+description: Use HTTP[S]_PROXY environment variables to define a proxy for the Azure SDK for .NET
+ms.date: 12/10/2020
+ms.topic: conceptual
+ms.custom: devx-track-dotnet
+---
+
+# Configure a proxy server when using the Azure SDK for .NET
+
+If your organization requires the use of a proxy server to access internet resources, you will need to set an environment variable with the proxy server information to use the Azure SDK for .NET.  
+
+## Configuration using environment variables
+
+Depending on if your proxy server uses HTTP or HTTPS, you will set either the environment variable HTTP_PROXY or HTTPS_PROXY respectively.  The proxy server URL has the form `http[s]://[username:password@]<ip_address_or_hostname>:<port>/` where the username:password combination is optional. To get the ip address or hostname, port and credentials for your proxy server, consult your network administrator.
```

Comment:
Code-fence these environment variables so that they aren't localized.

```suggestion
Depending on if your proxy server uses HTTP or HTTPS, you will set either the environment variable `HTTP_PROXY` or `HTTPS_PROXY` respectively. The proxy server URL has the form `http[s]://[username:password@]<ip_address_or_hostname>:<port>/` where the `username:password` combination is optional. To get the IP address or hostname, port and credentials for your proxy server, consult your network administrator.
```